src/main/java/com/walmartlabs/x12/util/ConversionUtil.java

```diff
@@ -0,0 +1,38 @@
+package com.walmartlabs.x12.util;
+
+import com.walmartlabs.x12.exceptions.X12ParserException;
+
+import java.math.BigDecimal;
+import java.math.RoundingMode;
+
+public final class ConversionUtil {
+
+    public static BigDecimal convertStringToBigDecimal(String theString, int decimalPlaces) {
+        BigDecimal returnValue = null;
+        try {
+            if (theString != null && theString.trim().length() > 0) {
+                returnValue = new BigDecimal(theString).setScale(decimalPlaces, RoundingMode.HALF_UP);
+            }
+        } catch (NumberFormatException e) {
+            throw new X12ParserException("Invalid numeric value");
+        }
+        return returnValue;
+    }
+
+    public static Integer convertStringToInteger(String theString) {
+        Integer returnInteger = null;
+
+        try {
+            if (theString != null && theString.trim().length() > 0) {
+                returnInteger = Integer.valueOf(theString);
+            }
+        } catch (NumberFormatException e) {
+            throw new X12ParserException("Invalid numeric value");
```

Comment:
Should we throw different exception and not `X12ParserException` because `ConversionUtil` is generic and we may not want to throw exception which is related to X12 Parsing. Thoughts?

Reply:
The primary purpose for `ConversionUtil` is to "contain" common methods used in parsing to convert values from a string to another data type. It is not expected to be used in other scenarios. I'm good w/ throwing this vs. catching and wrapping it with an `X12ParsingException` in every `parseXXX` method